db/migrate/20230405050133_create_posts.rb

```diff
@@ -1,7 +1,7 @@
-class CreatePost < ActiveRecord::Migration[7.0]
+class CreatePosts < ActiveRecord::Migration[7.0]
   def change
     create_table :posts do |t|
-      t.string :title
+      t.text :title
```

Comment:
- [ ] [OPTIONAL] I advise you to give the title a string data type because we just need a few texts.